ios/browser/api/brave_shields/adblock_engine.h

```diff
@@ -21,6 +21,7 @@ NS_SWIFT_NAME(AdblockEngine.MatchResult)
 @property(nonatomic, readonly) bool didMatchException;
 @property(nonatomic, readonly) bool didMatchImportant;
 @property(nonatomic, readonly) NSString* redirect;
+@property(nonatomic, readonly) NSString* rewritten_url;
```

Comment:
Just one minor nit, Obj-C doesn't usually use snake case like this, can we make this `rewrittenURL` instead? :) (and same below in adblock_engine.mm property decleration/setter)

Reply:
makes sense, updated!